bgs/bgs.go

```diff
@@ -136,6 +140,8 @@ func NewBGS(db *gorm.DB, ix *indexer.Indexer, repoman *repomgr.RepoManager, evtm
 	db.AutoMigrate(models.PDS{})
 	db.AutoMigrate(models.DomainBan{})
 
+	uc, _ := lru.New[string, *User](1_000_000)
```

Comment:
probably doesn't need to be that big, our DAU is only like... oh right, haha.

Reply:
yeah, this is our *hourly* active user count lately lol